docs/dev/lsp-extensions.md

```diff
@@ -376,12 +376,29 @@ rust-analyzer supports two `kind`s of runnables, `"cargo"` and `"shell"`. The `a
 
 ```typescript
 {
+    /**
+     * Environment variables to set before running the command.
+     */
+    environment?: Record<string, string>;
```

Comment:
Open to alternative names, should we be explicit with `environmentVariables`?

Reply:
could also call it `envVariables`?

Reply:
Eh, I'll stick with `environment` I think.

Reply:
fair enough! not terribly attached.